dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventRequestParams.java

```diff
@@ -133,27 +132,27 @@ public class EventRequestParams implements PageRequestParams {
   @OpenApi.Property({UID[].class, User.class})
   private Set<UID> assignedUsers = new HashSet<>();
 
-  private Date occurredAfter;
+  private StartDateTime occurredAfter;
 
-  private Date occurredBefore;
+  private EndDateTime occurredBefore;
 
-  private Date scheduledAfter;
+  private StartDateTime scheduledAfter;
 
-  private Date scheduledBefore;
+  private EndDateTime scheduledBefore;
 
   private StartDateTime updatedAfter;
 
   private EndDateTime updatedBefore;
 
   private String updatedWithin;
 
-  private Date enrollmentEnrolledBefore;
+  private EndDateTime enrollmentEnrolledBefore;
```

Comment:
I think it would make sense to group and order fields into Start then End. The type helps a lot as I was always wondering what is before/after mean again for the date range 😅